lighthouserc.yml

```diff
@@ -1,7 +1,30 @@
 ci:
   collect:
     numberOfRuns: 1
+    startServerCommand: sh ./lighthouse.sh
+    startServerReadyPattern: 'BUILD SUCCESSFUL'
+    startServerReadyTimeout: 600000
+    url:
+      - https://localhost.simplereport.gov/app
```

Comment:
Can you add a URL for the dashboard as well? The pattern should be
https://localhost.simplereport.gov/app/dashboard?facility=$FACILITY_ID

Reply:
Sure, can do! If it's just this one I don't mind adding it, for more urls I think we should add a new ticket, see comment below.

Reply:
Yep agreed! I just would like to see the dashboard added since it's the home page for admins and doesn't require any additional seed data, but would hold off on any other URLs.

Reply:
✅